package.json

```
@@ -67,7 +67,7 @@
     "@types/geojson": "^7946.0.16",
     "d3-format": "^3.1.0",
     "d3-scale": "^4.0.2",
-    "h3-js": "4.2.1",
+    "h3-js": "^4.2.1",
```

Outside of this PR scope, but it might be worth creating a ticket in `cloud-native`, so we align this dependency there, instead of bundling both versions 👍

---

deck.gl is using `4.1.0`, so can we do this instead? Otherwise we will be packaging 3 versions of h3 (cloud-native is on `3.7.2`)


```suggestion
    "h3-js": "^4.1.0",
```

---

Changed to ^4.1.0 here, and continued in:

- https://github.com/CartoDB/carto-kepler/pull/196